### PR TITLE
[IMPROVEMENT] Improved installation guide. 

### PR DESCRIPTION
**Pull request by T1duS**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [X] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that another pull request for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [X] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

{pull request content here}
This was a task of GCI - 2018.   
Improved the installation guide. Added a screencast demo.

### CODE REVIEW

File: install/installation.md [hunk 1]
```diff
@@ -26,53 +34,90 @@ cd /var/www/
...
 * Install XMing XServer and setup Putty for ssh connections.
-* Virt-manager can call ssh to make the connection to KVM Server and should be 
-able to run virsh and send commands to it. 
+* Virt-manager can call ssh to make the connection to KVM Server and should be
+able to run virsh and send commands to it.
 * Follow the steps from the Linux installation from within the Cygwin terminal
 to complete the platform's installation.
 
+## Screencast-Demo
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/hmNj3gbe5rM/0.jpg)](http://www.youtube.com/watch?v=hmNj3gbe5rM "CCExtractor - Sample Platform : Installation Tutorial")
```
Comment (canihavesomecoffee) [inline install/installation.md]: I really like the fact that you went to all the trouble, but it contains incorrect settings, and as I mentioned above, steps that should not be necessary. Can those things be fixed?

File: install/installation.md [hunk 2]
```diff
@@ -26,53 +34,90 @@ cd /var/www/
...
+sudo pip3 install -r requirements.txt
+```
+There might be an error something like libvirt not found. Ignore that
+```
+sudo apt-get install libvirt-dev
+```
+
+Running the program
+```
+sudo python3 run.py
```
[the remark this answers is not in the document]
Comment (MatejMecka) [reply install/installation.md]: Also using sudo is a bad idea.

[the remark this answers is not in the document]
Comment (T1duS) [reply install/installation.md]: For me, it gives permission denied without sudo.

[the remark this answers is not in the document]
Comment (satyammittal) [reply install/installation.md]: thats due to permission denied on log file. Give read, write permission to whole folder after downloading.

**Pull request closed without merging**